src/HW/client/src/redux/actions/constants/index.js

```diff
@@ -0,0 +1,13 @@
+const ACTIONS = {
+  SHOW_LOADER: "SHOW_LOADER",
+  HIDE_LOADER: "HIDE_LOADER",
```

Comment:
the first one should be START_FETCH
and the second can be part of the fetch successfully / failed action that will also pass the items if it was success